### PR TITLE
Various no-diff cleanups and debugging additions/fixes

### PR DESCRIPTION
**Pull request by BruceForstall**
1. Add debugging cVN/dVN dumpers of value numbers
2. Fix new loop debugging dumpers on x86
3. Display the bbID, not just bbNum, in more places
4. For ValueNum handle constants, display the handle constant type
5. Fix some grammar nits and try to rewrite some comments to read better

### CODE REVIEW

File: src/coreclr/jit/valuenum.cpp [hunk 1]
```diff
@@ -4017,66 +4017,66 @@ ValueNum ValueNumStore::EvalCastForConstantArgs(var_types typ, VNFunc func, Valu
             }
             break;
         }
-            {
```
Comment (BruceForstall) [inline src/coreclr/jit/valuenum.cpp:4020]: There's no code change here; just fixing the indenting by moving this open scope inside the 'case' below

File: src/coreclr/jit/valuenum.cpp [hunk 2]
```diff
@@ -8972,7 +8976,7 @@ void ValueNumStore::vnDump(Compiler* comp, ValueNum vn, bool isPtr)
                 }
                 else
                 {
-                    printf("LngCns: ");
```
Comment (BruceForstall) [inline src/coreclr/jit/valuenum.cpp:8975]: This is to be consistent with `IntCns`, above.

**Comment by BruceForstall** [comment]
I extracted these out of a change with semantic changes to make that one more focused.

**Comment by BruceForstall** [comment]
@AndyAyersMS PTAL
cc @dotnet/jit-contrib

File: src/coreclr/jit/gentree.cpp [hunk 3]
```diff
@@ -10710,6 +10709,57 @@ void GenTree::SetIndirExceptionFlags(Compiler* comp)
...
+        case GTF_ICON_BBC_PTR:
+            return printf("GTF_ICON_BBC_PTR");
+        case GTF_ICON_STATIC_BOX_PTR:
+            return printf("GTF_ICON_STATIC_BOX_PTR");
+        case GTF_ICON_FIELD_SEQ:
+            return printf("GTF_ICON_FIELD_SEQ");
+        case GTF_ICON_STATIC_ADDR_PTR:
+            return printf("GTF_ICON_STATIC_ADDR_PTR");
+        default:
+            assert(!"a forgotten handle flag");
```
Comment (AndyAyersMS) [inline src/coreclr/jit/gentree.cpp:10758]: Maybe not assert here? 

Slight preference for having methods like this just return strings, so they compose better into formatted output.

Also: we have both "Disp" and "dsp", should we settle on just one?

**Review by AndyAyersMS — APPROVED** [review]
Feel free to ignore my feedback or address later.

File: src/coreclr/jit/gentree.cpp (hunk 3 above)
Comment (BruceForstall) [reply src/coreclr/jit/gentree.cpp]: > Maybe not assert here?
> 
> Slight preference for having methods like this just return strings, so they compose better into formatted output.

Agreed, and fixed.

> Also: we have both "Disp" and "dsp", should we settle on just one?

Maybe? IMO, slight preference for "disp". Not sure it's worth the effort to change it.

**Comment by BruceForstall** [comment]
CI seems really broken

**Pull request merged**